node-api-dotnet/samples/NodeKernelSyntaxExamples/NodeKernelSyntaxExamples.csproj

```diff
@@ -1,19 +1,16 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <RootNamespace></RootNamespace>
     <TargetFramework>net6.0</TargetFramework>
     <RestorePackagesPath>$(MSBuildThisFileDirectory)/pkg</RestorePackagesPath>
     <OutDir>bin</OutDir>
     <NodeApiAssemblyJSModuleType>esm</NodeApiAssemblyJSModuleType>
+    <ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>None</ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>
+    <UserSecretsId>cd5a47f6-146f-41bf-8b77-427192d0ba0b</UserSecretsId>
```

Comment:
What is this for?

Reply:
- `<ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>` supresses this warning https://learn.microsoft.com/en-us/visualstudio/msbuild/errors/msb3270?view=vs-2022

- `<UserSecretsId>` is used by `dotnet secrets` as a way to store the the API Key using the Secret Manager tool